resources/views/components/input/checkbox.blade.php

```diff
@@ -1,7 +1,7 @@
 @props(['name' => '', 'id' => uniqid('checkbox-')])
 <label
     for="{{ $id }}"
-    {{ $attributes->only('class')->class('text-ct-primary relative flex cursor-pointer select-none flex-wrap items-center gap-x-3 text-sm') }}
+    {{ $attributes->only('class')->class('text-ct-primary relative flex cursor-pointer select-none flex-wrap gap-x-3 text-sm [&>span]:flex-1') }}
```

Comment:
a $slot->attributes->merge(['class'->'flex-1']) gave me errors so this would be the other solution, classes still merge onto the span but you cant predefine any of them on the span